lib/galaxy/tool_util/xsd/galaxy.xsd

```diff
@@ -2393,9 +2393,25 @@ In your command template, you'll need to include the section name to access the
 variable:
 
 ```
---color $adv.plot_color
+$section_name.parameter_name
 ```
 
+In output filters sections are represented as dictionary with the same name as the section:
+
+```
+<filter>section_name['parameter_name']</filter>
+```
+
+In order to reference parameters in sections from tags in the `<outputs>` section, e.g. in the `format_source` attribute of `<data>` tags, the syntax is:
+
+```
+section_name|parameter_name
```

Comment:
can you give an example with format_source here?

Reply:
sure